content/dogepedia/articles/charities-accepting-doge.md

```diff
@@ -3,6 +3,10 @@ title = "What are some charities that accept Dogecoin?"
 date = "2021-10-22"
 [ author ]
   name = "Dogecoin"
-+++
++++]
+
+Dogecoin’s history includes many charitable endeavours: from funding clean water wells in 2014 for Kenya, to the #TeamSeas initiative in 2021 – and many worthwhile causes in-between. The past year, though, has seen a dramatic increase in charities directly accepting Dogecoin donations – which is very much wow! 
+
+Organisations like *The Giving Block* make it super easy for charities to accept Dogecoin and other cryptocurrency donations. Initiatives like *The Water Project* — which has been accepting Dogecoin for over 6-years — now use *The Giving Block* to accept Dogecoin donations to fund fully monitored water projects in Africa. The *American Cancer Society* is also another charity that utilises this service, and uses Dogecoin donations to help patients access lifesaving care and resources, as they work toward the goal of ridding the world from cancer.  
 
-Lists to charities we can recommend accepting Doge.
\ No newline at end of file
+There are hundreds of extremely important and worthwhile charities through which you can donate some Dogecoin. If you’re passionate about a cause and a charity's mission, then #DoOnlyGoodEveryday and donate some Doge!
```

Comment:
maybe change 'you can donate some Dogecoin.' to 'you can donate Dogecoin.' and then on the last 'donate some Doge!' change to 'donate some Dogecoin!'?